ors-engine/src/main/java/org/heigit/ors/matrix/MatrixRequest.java

```diff
@@ -224,9 +222,9 @@ else if (getSearchParameters().getDynamicSpeeds() && routingProfile.getGraphhopp
         } catch (PointNotFoundException e) {
             throw e;
         } catch (MaxVisitedNodesExceededException e) {
-            throw new InternalServerException(MatrixErrorCodes.MAX_VISITED_NODES_EXCEEDED, "Unable to compute a distance/duration matrix: " + e.getMessage());
+            throw new StatusCodeException(500, MatrixErrorCodes.MAX_VISITED_NODES_EXCEEDED, "Unable to compute a distance/duration matrix: " + e.getMessage());
```

Comment:
Am I missing something here, or is this just a refactoring? According to 

https://github.com/GIScience/openrouteservice/blob/6cf11dbc2cc12fdc1191f2eab06c50f07cd18840/ors-engine/src/main/java/org/heigit/ors/exceptions/InternalServerException.java#L29-L31

a call to `StatusCodeException(500, MatrixErrorCodes.MAX_VISITED_NODES_EXCEEDED, "...")` is effectively the same as `InternalServerException(MatrixErrorCodes.MAX_VISITED_NODES_EXCEEDED, "...")`.